pkgs/modules/default.nix

```diff
@@ -11,12 +11,18 @@ let
       pypkgs = pkgs.python310Packages;
     }))
     (mkModule ./pyright-extended)
+
     (mkModule (import ./nodejs {
       nodejs = pkgs.nodejs-18_x;
     }))
     (mkModule (import ./nodejs {
       nodejs = pkgs.nodejs_20;
     }))
+
+    (mkModule (import ./svelte-kit {
```

Comment:
You should be able to do just `(mkModule ./svelte-kit)` and it will have access to pkgs-unstable. Look at the rust module.